fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -1020,6 +1020,7 @@ public void initialize(String[] args) throws Exception {
         // 3. Load image first and replay edits
         this.editLog = new EditLog(nodeName);
         loadImage(this.imageDir); // load image file
+        this.workloadGroupMgr.appendInternalWorkloadGroup();
```

Comment:
Not call this method in load image method. We should isolate workload manager's init only in workload mgr.